packages/drivers/odsp-driver/src/createFile.ts

```diff
@@ -91,7 +93,10 @@ export async function createNewFluidFile(
 
     const odspUrl = createOdspUrl({ ... newFileInfo, itemId, dataStorePath: "/" });
     const resolver = new OdspDriverUrlResolver();
-    const odspResolvedUrl = await resolver.resolve({ url: odspUrl });
+    const odspResolvedUrl = await resolver.resolve({
+        url: odspUrl,
+        headers: { [ClpCompliantAppHeader.isClpCompliantApp]: isClpCompliantApp },
```

Comment:
If `isClpCompliantApp` is `undefined` should this header get the value `undefined` or `false`?

Reply:
Also, existing usage in `epochTracker` does `toString()`, feels like the two places should match.

Reply:
Both will have the same effect as long as it is not true.

Reply:
No, The one in epoch tracker is added in headers of network apis while here it is just set in the resolved url.